src/modules/75-cf/pages/environment-details/__tests__/EnvironmentDetails.test.tsx

```diff
@@ -64,71 +92,84 @@ describe('EnvironmentDetails', () => {
       useSyncedEnvironment: () => ({ data: mockEnvironment, loading: undefined, error: undefined, refetch: jest.fn() })
     })
 
-    render(
-      <TestWrapper
-        path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/onboarding/detail"
-        pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
-      >
-        <EnvironmentDetails />
-      </TestWrapper>
-    )
+    renderComponent()
 
     expect(getAllByText(document.body, mockEnvironment.data.name)).toBeDefined()
     expect(getAllByText(document.body, mockEnvironment.data.identifier)).toBeDefined()
 
     expect(getAllByText(document.body, mockApiKeys.apiKeys[0].name)).toBeDefined()
-    expect(getAllByText(document.body, mockApiKeys.apiKeys[1].name)).toBeDefined()
   })
 
-  test('CFEnvironmentDetailsBody should render loading', async () => {
+  test('should render toolbar when there are existing SDK Keys ', async () => {
+    mockImport('services/cd-ng', {
+      useGetEnvironment: () => ({ data: mockEnvironment, loading: false, error: undefined, refetch: jest.fn() })
+    })
+
     mockImport('services/cf', {
       useGetAllAPIKeys: () => ({ data: mockApiKeys })
     })
 
-    render(
-      <TestWrapper
-        path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/onboarding/detail"
-        pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
-      >
-        <EnvironmentDetailsBody environment={mockEnvironment.data as EnvironmentResponseDTO} />
-      </TestWrapper>
-    )
+    renderComponent()
 
-    expect(document.body.querySelector('[class*=ContainerSpinner]')).toBeDefined()
-  })
+    const toolbar = document.getElementsByClassName('PageSubHeader--container')[0]
 
-  test('CFEnvironmentDetailsBody should render error', async () => {
-    mockImport('services/cf', {
-      useGetAllAPIKeys: () => ({ error: { message: 'SOME ERROR' } })
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: 'cf.environments.apiKeys.addKeyTitle' })).toBeInTheDocument()
+      expect(toolbar).toBeInTheDocument()
```

Comment:
```suggestion
      expect(document.getElementsByClassName('PageSubHeader--container')[0]).toBeInTheDocument()
```
You shouldn't have to const this if it is just being used in one place, it's personal preference though :)

Reply:
sure thang